x-pack/legacy/plugins/index_management/public/app/sections/home/index_list/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -22,10 +22,21 @@ import {
   EuiOverlayMask,
   EuiCheckbox,
 } from '@elastic/eui';
+
+// We will be able to remove these after the NP migration is complete.
+import { toastNotifications, fatalError } from 'ui/notify';
+import { createUiStatsReporter } from '../../../../../../../../../../src/legacy/core_plugins/ui_metric/public';
+import { getNewPlatformCompatibleHttpClient } from '../../../../services/api';
+
 import { flattenPanelTree } from '../../../../lib/flatten_panel_tree';
 import { INDEX_OPEN } from '../../../../../../common/constants';
 import { getActionExtensions } from '../../../../../index_management_extensions';
-import { getHttpClient } from '../../../../services/api';
+
+// We will be able to remove this after the NP migration is complete and we can just inject the
+// NP fatalErrors service..
+const getNewPlatformCompatibleFatalErrorService = () => ({
```

Comment:
@sebelga Here's another example of a service adapter that will become unnecessary as part of your migration.